src/interp.rs

```diff
@@ -0,0 +1,143 @@
+use parse::Bytecode;
+use parse::Instr;
+use std::collections::HashMap;
+
+static GLOBAL_NSPACE: &'static str = "global";
+static MAIN_FN: &'static str = "main";
+
+#[derive(Clone)]
+enum NativeType {
+    Int(i32),
+    Double(f32),
+    Bool(bool),
+    Str(String),
+    ObjectRef(usize),
+}
+
+#[derive(Clone)]
+struct Object {
+    class_name : String,
+    fields: HashMap<String, NativeType>,
+}
+
+impl Object {
+    fn new(class_name: String) -> Object {
+        Object {
+            class_name: class_name,
+            fields: HashMap::new()
+        }
+    }
+
```

Comment:
Extraneous blank line.

Reply:
Think this was fixed as part of 4a77e577d8c76096f154c27dd1c13b14fbd68e4b